app/app-controllers/error-screen/errorController.js

```diff
@@ -73,24 +80,37 @@ define([
 
     ErrorController.prototype._generateErrorCallback = function(errorType, params) {
         var self = this;
+        var navigator = params.navigator;
         var backHandler = params.backHandler;
         var retryHandler = params.retryHandler;
         var isActiveItem = params.isActiveItem;
+        var canGoBack = params.canGoBack;
 
         return function(eventArgs) {
+            navigator.loaded = false;
+
             if (isActiveItem()) {
                 self.viewPlugin.once('back', function() {
                     self.hide();
                     self._removeModalEvents();
                     backHandler();
                 });
+
+                // Wait until the error page is loaded before showing
+                self.viewPlugin.on('error:loaded', function() {
```

Comment:
I'd love to see a standardized event for pages to signal Astro apps that they're done loading.  Could we maybe refactor to use `astro:page-loaded` or something like that?   

#blue-sky -> Be cool to push a PR into Adaptive and Progressive projects that triggered that event somewhere automatically.